app-misc/piper/piper-0.5.1-r2.ebuild

```
@@ -40,6 +40,8 @@ DEPEND="
 	virtual/libudev
 "
 
+PATCHES=( "${FILESDIR}"/${P}-disable-flake8-linting.patch )
```

nitpick but even for 1 patch I tend to prefer patch on its own line (makes it quicker to add/remove patches without restyling everytime), aka:
```suggestion
PATCHES=(
	"${FILESDIR}"/${P}-disable-flake8-linting.patch
)
```